Client.xcodeproj/project.pbxproj

```diff
@@ -19812,10 +19824,13 @@
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_ENTITLEMENTS = "";
+				CODE_SIGN_IDENTITY = "";
+				CODE_SIGN_STYLE = Manual;
```

Comment:
Any reason for changing this now? I guess manual is indeed safer but this is not related to Snapshot Testing is it?

`PROVISIONING_PROFILE_SPECIFIER = "";` below also seems a bit strange 🤔

Reply:
These are all part of the `EcosiaSnapshotTests` specific target. 
We do run the tests passing those values specifically at CI level as well, but if we do not update those for the target specifically, we will have issues on running the tests locally every time.